packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -353,14 +353,16 @@ export default class TestConfiguration {
       roles?: UserRoles
     } = {}
   ): Promise<User> {
-    let { id, firstName, lastName, email, builder, admin, roles } = user
-    ;(firstName = firstName || generator.first()),
-      (lastName = lastName || generator.last()),
-      (email = email || generator.email())
-    roles = roles || {}
-    if (builder == null) {
-      builder = true
-    }
+    const {
+      id,
+      firstName = generator.first(),
+      lastName = generator.last(),
+      email = generator.email(),
+      builder = true,
```

Comment:
TIL!

```javascript
> obj = {"foo": 1, "bar": 2}
{ foo: 1, bar: 2 }
> const { foo, bar = 3 } = obj
undefined
> foo
1
> bar
2
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value
You can define defaults to all destructures